en/components/grids_templates/column-resizing.md

```diff
@@ -287,7 +287,7 @@ $custom-grid-theme: igx-grid-theme(
 ```
 
 >[!NOTE]
->The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please, refer to [`Palettes`](../themes/palette.md) topic for detailed guidance on how to use them.
+>The `igx-color` and `igx-palette` are powerful functions for generating and retrieving colors. Please, refer to [`Palettes`](../themes/palettes.md) topic for detailed guidance on how to use them.
```

Comment:
I believe this link should be "../themes/sass/palettes.md"

Reply:
changed